site/_docs/reference.md

```diff
@@ -2681,6 +2681,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | CEIL(value)                                    | Similar to standard `CEIL(value)` except if *value* is an integer type, the return type is a double
 | m s | CHAR(integer)                                | Returns the character whose ASCII code is *integer* % 256, or null if *integer* &lt; 0
 | b o p | CHR(integer)                               | Returns the character whose UTF-8 code is *integer*
+| b | CODE_POINTS_TO_BYTES(array&lt;integer&gt;)     | Converts an *array&lt;integer&gt;* of extended ASCII code points to bytes, return null if array is null or the element of the array is null
```

Comment:
I would call the argument `integers`, and describe it 'Converts *integers*, an array of integers between 0 and 255, into a byte string'. `array<integer>` is not how Calcite writes types.

I would mention that it throws if any element is not between 0 and 255 inclusive; no need to mention that it returns null if `integers` is null.